app/views/content/is-teaching-right-for-me/if-you-want-to-change-career.md

```diff
@@ -7,7 +7,7 @@ description: |-
 related_content:
```

Comment:
@emmaachesongray Bit confused why this file is still in ITRFM - shouldn't it have been moved to LAAT rather than this file still existing and a new duplicate file having been created?

Reply:
Confused as well as I think I only moved them, not even sure how to duplicate a file! Mightve been from resolving merge conflicts? Will tidy up either way!